src/marionnette_extensions/page.coffee

```diff
@@ -7,10 +7,11 @@ class Page extends Marionette.LayoutView
   canGoBack: ->
     @_parent.canGoBack()
 
-  transitionBack: (route, options) ->
+  transitionBack: (route, options = {}) ->
```

Comment:
Actually the options hash is available to the show method, because the hash change doesn't reload the page and the `AnimatableRegion` is a singleton.